README.md

```diff
@@ -194,6 +194,14 @@ Steps:
 4) Run `./hack/development-mode.sh` or `./hack/preview.sh`
 5) Trigger update in ArgoCD and delete `application-service-controller-manager` pod manually or run `oc rollout restart -n application-service deployment/application-service-controller-manager`
 
+## Authentication
+
+Authentication is managed by `components/authentication`. Authentication is disabled in development modes.
+
+For access to Stage cluster the github user has to be part of `stage` team in `redhat-appstudio-sre` organization.
```

Comment:
would it make sense to add an info here about who to contact to get an access to the stage cluster?

Reply:
added